rados/snapshot_test.go

```diff
@@ -0,0 +1,23 @@
+package rados_test
+
+import (
+	"testing"
+
+	"github.com/immesys/go-ceph/rados"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSnapshots(t *testing.T) {
+	conn, _ := rados.NewConn()
+	conn.ReadDefaultConfigFile()
+	conn.Connect()
+	defer conn.Shutdown()
+
+	pool_name := GetUUID()
+	err := conn.MakePool(pool_name)
```

Comment:
how does this work? What rule does it use? what placement groups? Interesting.

BTW you can test this on devmachine if you use dvcephgetconf

Reply:
yeh, zero idea - it's what the other tests were doing … but clearly they have some other setup that I don't have. Although it did pass on stratus ceph - do you think I messed something up?